src/Illuminate/Foundation/Console/OptimizeCommand.php

```diff
@@ -96,19 +100,22 @@ protected function compileClasses()
     }
 
     /**
-     * Get the node traverser used by the command.
+     * Get the class preloader used by the command.
      *
-     * @return \ClassPreloader\Parser\NodeTraverser
+     * @return \ClassPreloader\ClassPreloader
      */
-    protected function getTraverser()
```

Comment:
> If someone installs 2.x, the optimize command works just fine. If someone installs 3.x, it also works, but we use the new factory class rather than building the traverser ourselves.

If someone install 2.x, they can still use `getTraverser()` method, if someone install 3.x, use factory class. Even if `getTraverser()` not used all the time, it will be used in some requirement.